index.md

```diff
@@ -4,61 +4,38 @@ root: .  # Is the only page that doesn't follow the pattern /:path/index.html
 
-As a real-life example of the above scenario, let me describe various computing components of the HPC that I am using at ICHEC. The most recent HPC implementation at ICHEC is known as [Kay](https://www.ichec.ie/about/infrastructure/kay). Kay has four key components:
-
-| component nanme | processing units | How many of them? | cores/node |
-|-----------------|------------------|-------------------|---------|
-| cluster | Intel Xeon Gold (Skylake)/2.4 GHz/192 GiB RAM | 336 nodes | 40 |
-| High memory | Intel Xeon Gold (Skylake)/2.4 GHz/1 TiB RAM | 6 nodes | 40 |
-| Phi | Intel Xeon Phi 7210 (KNL)/ 1.3 GHz/ 192 GiB RAM | 16 nodes | 40 |
-| GPU | Skylake + 2X NVIDIA Tesla V100 16GB PCI | 16 nodes | 5120 CUDA cores & 640 Tensor cores |
-
-It is evident that these components would work differently and therefore a software performance will vary depending on what component it is being run, and how optimized the code is for that platform. By now, you might have some realization about how different it is to run a LAMMPS job on a laptop or Desktop from running it on a HPC!
-
-> ## Well, let us know a little more about your HPC processing units now
->Take a minute and write down the computing units that your HPC is providing you.
+On any machine, with different components, they would work differently and therefore a software's performance will vary depending on what component it is being run on, and how optimized the code is for that platform. There are no such complications on a standard desktop or laptop, running on an HPC is very, very different.  
 
-> ## Have you ever used GPU or Phi to run LAMMPS?
-> * Write a few sentences about which components/processors are you running your LAMMPS jobs?
-> * Probably, a more enagaing one: Do you know whether your LAMMPS binary was prepared to run optimally on this components?
+> ## Note
+> 
+> - This is the draft HPC Carpentry release. Comments and feedback are welcome.
+{: .callout}
 
 > ## Prerequisites
-
-> For these lessons, I'll assume that you have a prior experience on working in a HPC cluster, basic Linux/unix commands, shell scripts, and the primary knowledge to understand LAMMPS inputs files and running LAMMPS jobs in a cluster.
-
-> To know about the basic LAMMPS commands, you may go through this [link](https://lammps.sandia.gov/doc/Commands_all.html).
-
-> To get a basic introduction on HPC, you may follow [this](https://github.com/hpc-carpentry/hpc-intro) link.
-{:.prereq}
+>
+> - Command line experience is required to take this course as well as shell scripting.
+> - Prior experience working on a HPC cluster. If you are new to a HPC cluster we recommend participants to go through [hpc-intro](https://hpc-carpentry.github.io/hpc-intro/).
+> - You should be familiar with working with LAMMPS and understand its input files and be able to run LAMMPS jobs in an HPC cluster.
```

Comment:
They don't need to be able to run LAMMPS jobs on an HPC system...that is what we are supposed to be teaching them!